shared/text.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Maybe let's put some structure in this json? Soon it'll become very messy.

Reply:
And similar suggestion as for theme - maybe we could have it provided in some context wrapper for whole application? I think it'd look very clean :)